lib/popcorn/pop_validation.nit

```diff
@@ -516,6 +516,50 @@ class FloatField
 	end
 end
 
+# Check if a field is a Bool
+#
+# ~~~
+# var validator = new ObjectValidator
+# validator.add new BoolField("field", required=false)
+# assert validator.validate("""{}""")
+# assert validator.validate("""{ "field": true }""")
+# assert validator.validate("""{ "field": false }""")
+# assert not validator.validate("""{ "field": "foo" }""")
+#
+# validator = new ObjectValidator
+# validator.add new BoolField("field")
+# assert not validator.validate("""{}""")
+# assert validator.validate("""{ "field": true }""")
+# assert validator.validate("""{ "field": false }""")
+# assert not validator.validate("""{ "field": "foo" }""")
+#
+# No type conversion is applied on the input value:
```

Comment:
comments in docunits should be outside the block to be markdown rendered

~~~
# [...]
# assert not validator.validate("""{ "field": "foo" }""")
￼# ~~~
#
# No type conversion is applied on the input value:
#
# ~~~
# assert not validator.validate("""{ "field": "true" }""")

~~~